### ISSUE TITLE
Build with Maven 3.8+ fails

### ISSUE THREAD

**Issue by AB-xdev** [xdev-software/biapi#25]
https://github.com/xdev-software/biapi/runs/2716112080?check_suite_focus=true

The build with Maven 3.8+ fails, because of 
```
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.10.0/jackson-databind-2.10.0.pom (7.1 kB at 393 kB/s)
[INFO] Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.10.0/jackson-annotations-2.10.0.pom
[INFO] Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.10.0/jackson-annotations-2.10.0.pom (2.7 kB at 125 kB/s)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.321 s
[INFO] Finished at: 2021-06-01T07:03:03Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal on project biapi: Could not resolve dependencies for project com.xdev-software:biapi:jar:5.1.1-SNAPSHOT: Failed to collect dependencies at net.sf.jasperreports:jasperreports:jar:6.16.0 -> com.lowagie:itext:jar:2.1.7.js8: Failed to read artifact descriptor for com.lowagie:itext:jar:2.1.7.js8: Could not transfer artifact com.lowagie:itext:pom:2.1.7.js8 from/to maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories: [jaspersoft-third-party (http://jaspersoft.jfrog.io/jaspersoft/third-party-ce-artifacts/, default, releases+snapshots), jr-ce-releases (http://jaspersoft.jfrog.io/jaspersoft/jr-ce-releases, default, releases+snapshots)] -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
Error: Process completed with exit code 1.
```

**Comment by AB-xdev** [comment]
Underlying problem was fixed with 
https://github.com/TIBCOSoftware/jasperreports/commit/8c0eb532f1731935d0d7bd69daea7c7cb7e14783

https://github.com/xdev-software/biapi/pull/16 should fix this issue

**Comment by AB-xdev** [comment]
Fixed by #16
